lib/Prometheus/Tiny.pm

```diff
@@ -5,7 +5,8 @@ package Prometheus::Tiny;
 use warnings;
 use strict;
 
-use Carp qw(croak);
+use Carp qw(croak carp);
+use Scalar::Util qw(looks_like_number);
```

Comment:
`looks_like_number` is kinda weird, in that things like `Inf` are numbers (and it also doesn't work correctly on tied variables, lol), but I think it's probably fine, since you don't really want to be in the business of writing a regex for all valid numeric inputs.

Reply:
Actually I _think_ `Inf` might actually be valid for metrics lol. But yeah, I just wanted something that would do slightly more than just a regular definedness check. If someone comes in with something weird and/or a tie, I'll sort it then.